app/src/constants/Constants.ts

```diff
@@ -29,3 +32,18 @@ export const NANOSECONDS_IN_ONE_DAY = NANOSECONDS_IN_ONE_HOUR * 24;
 
 export const NUM_DATA_POINTS_RATIO = 0.5;
 export const ANIMATION_TIME = 500;
+
+export const INSTR_COLOR = {
+    AAPL: {
+        backgroundColor: Colors.indigo,
+    },
+    MSFT: {
+        backgroundColor: Colors.emerald,
+    },
+    SPY: {
+        backgroundColor: Colors.orange,
+    },
+    default: {
+        backgroundColor: Colors.grey,
+    },
+};
```

Comment:
Instruments shouldn't be hard coded, they should be fetched on the backend

Reply:
If we want the tickers to always have the same color, how else can we proceed?

Reply:
i think saving the instruments to redux would be a good way to serve all our use cases.

Reply:
the fetched instruments, that is